frontend/components/ActivityMap.vue

```diff
@@ -0,0 +1,248 @@
+            (activity: Activity) => activity.latitude && activity.longitude,
+        );
+
+        activitiesWithLocation.value.forEach((activity: Activity) => {
+            useMapboxPopup(activity.id, (popup) => {
+                popup.remove();
+            });
+
+            useMapboxMarker(activity.id, (marker) => {
+                const markerEl = marker.getElement();
+                const elements = markerEl.querySelectorAll("svg path");
+                elements.forEach(function (element) {
+                    if (element.getAttribute("fill")) {
+                        element.setAttribute("fill", markerColor(activity));
+                    }
+                });
+            });
+        });
+
+        activitiesWithoutLocation.value = activities.value.filter(
+            (activity: Activity) => !activity.latitude || !activity.longitude,
+        );
+    },
+    { immediate: true },
+);
+
+const colorMode = useColorMode();
+const darkTheme = window.matchMedia("(prefers-color-scheme: dark)");
+
+const colorAdded = fullConfig.theme.accentColor["input-label"] as string;
+const colorNotAdded = fullConfig.theme.accentColor[
+    "marker-not-added"
+] as string;
+
+function markerColor(activity: Activity) {
+    return activity.calendar_activities?.length > 0
+        ? colorAdded
+        : colorNotAdded;
+}
+
+const lat = computed(() => journey.getLat());
+const long = computed(() => journey.getLong());
+const zoom = computed(() => ((long.value || lat.value) === null ? 1 : 8));
+const style = computed(() =>
+    colorMode.preference === "dark" ||
+    (darkTheme.matches && colorMode.preference === "system")
+        ? "mapbox://styles/mathematti/clw4z6v0s028p01o0askbhfh9"
+        : "mapbox://styles/mathematti/clw4znxzh02mh01qz5hgk3qb6",
+);
+</script>
+
+<template>
+    <div class="flex items-center justify-center md:justify-start">
+        <div
+            class="relative mt-5 flex w-[90%] items-center sm:w-5/6 md:ml-[10%] md:w-[calc(50%+16rem)] md:justify-between lg:ml-10 lg:w-[calc(33.33vw+38.5rem)] xl:ml-[10%] xl:w-[calc(33.33vw+44rem)]"
+        >
+            <div class="-mb-2.5 text-2xl font-semibold">
+                <T key-name="journey.map" />
+            </div>
+            <span
+                v-if="activitiesWithoutLocation.length > 0"
+                v-tooltip.top="{
+                    value: t('journey.map.notfound.tooltip'),
+                    pt: { root: 'font-nunito' },
+                }"
+                class="pi pi-exclamation-circle -mb-2.5 ml-auto text-xl text-cancel-border hover:cursor-pointer"
+                @click="isNotFoundActivitiesDialogVisible = true"
+            />
+        </div>
+    </div>
+    <div class="flex justify-center md:justify-start">
+        <div
+            class="relative mt-5 flex h-96 w-[90%] items-end sm:w-5/6 md:ml-[10%] md:w-[calc(50%+16rem)] md:justify-start lg:ml-10 lg:w-[calc(33.33vw+38.5rem)] xl:ml-[10%] xl:w-[calc(33.33vw+44rem)]"
```

Comment:
Kannst die Map in der Handy Version kleiner machen? Also die Höhe? Die neue Höhe sollte zumindest die hälfte von der höhe der jetzigen Karte